Makefile

```diff
@@ -83,11 +83,50 @@ godoc: build
 
 .PHONY: godoc
 
+# --- OCI image ----------------------------------------------------------------
+
+# OCI builds can target different environments by defining the platform,
+# container registry and image tag for the environment and setting OCI_BUILD
+# to the environment name. The "release" environment is pre-defined for
+# releasing formal OCI images. The default null environment produces an image
+# for the local image store, local architecture with no image tag (which means
+# "latest" to Docker).
```

Comment:
I can't really follow this. I vaguely understand what's going on, but another example for how'd you'd run this with another OCI_BUID would have gone a long way.
In general I find this variable names constructed of other variable names really hard.

but go ahead anyway - this does what you want it to do, so it is good.